bridge/cypress/integration/oauth.spec.ts

```diff
@@ -0,0 +1,30 @@
+import BasePage from '../support/pageobjects/BasePage';
+
+describe('Test OAuth', () => {
```

Comment:
Can you please also add test cases for the other cases:
* Auth type BASIC
* Incorrect API Key

Reply:
Added.

Reply:
Actually, the incorrect API key isn't tested yet and I'm sure @laneli meant that the test should be inside the UI test instead of the unit tests.

Reply:
Right, I adjusted the unit test and also added UI tests for the BASIC auth and incorrect api key cases